core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -1097,7 +1100,9 @@ private[spark] object JsonProtocolSuite extends Assertions {
       |  },
       |  "Task Metrics": {
       |    "Executor Deserialize Time": 300,
+      |    "Executor Deserialize CPU Time": 0,
```

Comment:
Could you use values other than `0` so that you're sure the code is actually parsing the value, instead of falling into the default case?